.travis.yml

```diff
@@ -0,0 +1,22 @@
+dist: xenial
+language: python
+python:
+  - "3.6"
```

Comment:
Where are we only validating 3.6? MindMeld supports 3.4-3.7

Reply:
We were running into some issues with Travis CI for other versions
https://travis-ci.org/cisco/mindmeld/builds/556563795?utm_source=github_status&utm_medium=notification